docs/REST API/rest-api.md

```diff
@@ -8298,6 +8315,13 @@ _Available in Fleet Premium_
         "host_batch_size": 0
       }
     },
+    "integrations": {
+      "google_calendar": {
```

Comment:
@rachaelshaw @getvictor What API field does the "[Resolution webhook url"](https://www.figma.com/file/p81nWodxL04YD7iNyr9xVa/%2317230-Fleet-in-your-calendar?type=design&node-id=513%3A9265&mode=dev) form field translate to? Are we missing a field here?

Reply:
Maybe the below `email` field should be `resolution_webhook_url`?

Reply:
So when setting, `policies` subfields can be `name` or `id`, but the API always returns `policy_ids: [1,2,3]` right?

Reply:
Can we talk about this more @ standup? I want to understand why a `policy_ids` parameter won't work here. (It's more consistent w/ other parts of the API if we're able to accept an array of numbers here instead of an array of objects.)

Reply:
> It is `webhook_url`.

Looks like we call it `destination_url` for failing policy automations, but definitely agree `webhook_url` is better here, since the calendar integration could potentially have other options in the future besides triggering a webhook.